deployment/aks-periscope.yaml

```diff
@@ -140,7 +140,7 @@ metadata:
   name: clustertype-config
   namespace: aks-periscope
 data:
-  CLUSTER_TYPE: # <custom flag>
+  CLUSTER_TYPE: "managedCluster" # <custom flag>
```

Comment:
Why did we decide to change the clusterType value back to managerCluster value? Is this for testing purposes?

Reply:
This seems like the only way we can move forward for the 1-yaml deployment. I was keen to take complexity away from your scenario for Arc and avoid multiple point of entry for deployment.

If there is no default the deployment will fail with message like: cannot pass nil to vat cluster_type.

I think we could use this well but this will help your Arc PR not to become any complicated.

Reply:
Sounds good, let's leave the clusterType value as is then.